pandas/compat/__init__.py

```diff
@@ -12,9 +12,10 @@
 import os
 import platform
 import sys
-from typing import TYPE_CHECKING
 
 from pandas._typing import F
+import pandas.compat._compressors
+from pandas.compat._constants import *  # noqa: F401, F403
```

Comment:
Would prefer explicitly listening all the constants

Reply:
Sounds good. Done in the recent commit. Please take a look :)